projects/github-actions/test-results-to-slack/src/slack.js

```diff
@@ -156,6 +156,41 @@ async function getMessage( client, channelId, identifier ) {
 	return message;
 }
 
+/**
+ * Uploads a file to Slack using filesUploadV2.
+ *
+ * @param {object} client     - Slack client instance
+ * @param {string} filePath   - Path to the file to upload
+ * @param {string} channel_id - Channel ID where the file will be shared
+ * @param {string} thread_ts  - Thread timestamp for threading the file upload
+ * @return {Promise<object>} The response from the Slack API
+ */
+async function uploadFileToSlack( client, filePath, channel_id, thread_ts ) {
+	try {
+		const filename = path.basename( filePath );
+
+		if ( ! fs.existsSync( filePath ) ) {
+			throw new Error( `File not found: ${ filePath }` );
+		}
+
+		const response = await client.filesUploadV2( {
+			channel_id,
+			thread_ts,
+			file: filePath,
+			filename,
+		} );
```

Comment:
I wonder why you don't just inline this above like the previous code did.

The file existence check on line 172 is redundant to the one on line 27.

Reply:
Yes, now that I got rid of the extra complexity and use filesUploadV2, we can revert back to what it was. 👍 Done in fdea35ffa8cf7e88861cb85e251c07c68720763d